watertap/unit_models/reverse_osmosis_1D.py

```diff
@@ -1156,21 +1156,26 @@ def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
         # setting scaling factors for variables
         for j in self.config.property_package.component_list:
-            iscale.set_scaling_factor(self.permeate_side[0, 0].flow_mass_phase_comp['Liq', j], 0)
+            iscale.set_scaling_factor(self.permeate_side[0, 0].flow_mass_phase_comp['Liq', j], 1e+5)
```

Comment:
This scaling factor used to be 0? That seems like that should have been a big issue, I'm surprised it worked before.

Reply:
I think this was because the inlet was skipped.